drizzlepac/runastrodriz.py

```diff
@@ -453,29 +452,36 @@ def process(inFile, force=False, newpath=None, num_cores=None, inmemory=True,
                                              force_alignment=force_alignment,
                                              find_crs=True, **adriz_pars)
 
-        if align_with_apriori:
-            _trlmsg = _timestamp('Starting alignment with a priori solutions')
-            _trlmsg += __trlmarker__
-            if align_dicts is not None:
-                find_crs = not align_dicts[0]['alignment_verified']
-            else:
-                find_crs = False
+        if align_dicts is not None:
+            find_crs = not align_dicts[0]['alignment_verified']
+        else:
+            find_crs = False
+
+        # run updatewcs with use_db=True to insure all products have
+        # have a priori solutions as extensions
+        # This is NOT done in the apriori sub-directory in order to insure that all
+        # relevant solutions from the astrometry database get appended to the input FLT/FLC files.
+        #
+        updatewcs.updatewcs(_calfiles, use_db=True, all_wcs=False)
+        if _calfiles_flc:
+            _trlmsg += "Adding apriori WCS solutions to {}\n".format(_calfiles_flc)
+            updatewcs.updatewcs(_calfiles_flc, use_db=True, all_wcs=False)
```

Comment:
Should there be a message the for the 'a priori' solutions being added to the _calfiles in addition to the the message for the _calfiles_flc just so the user does not have to assume it is being done to the flt by default?